Content.Server/_DEN/TimedDespawn/TimedDespawnDetailSystem.cs

```diff
@@ -0,0 +1,107 @@
+using Content.Shared._DEN.TimedDespawn;
+using Content.Shared.Examine;
+using Robust.Server.Audio;
+using Robust.Server.GameObjects;
+using Robust.Shared.Timing;
+
+
+namespace Content.Server._DEN.TimedDespawn;
+
+
+/// <summary>
```

Comment:
Remove extra blank lines
```suggestion
using Robust.Shared.Timing;

namespace Content.Server._DEN.TimedDespawn;

/// <summary>
```